cdbot/cogs/cyber.py

```diff
@@ -167,10 +167,18 @@ async def flag(
             content = "13.1 is a No Flag Zone™ 🙅⛔⚔️"
         else:
             # Generates random, but unique and identical per challenge, base 64 "flag"
-            content = (
-                "The flag is:"
-                f"||{await generatebase64(ord(base[0]) + level_num + challenge_num)}||"
-            )
+            if random.randint(1, 5) == 5:
+                content = (
```

Comment:
Content is part of an embed. I know you can't put videos in embeds, but just posting the link to it within an embed would look rubbish. Maybe drop the embed entirely if this happens, so Discord expands the link?

Reply:
Just `ctx.send(content)` work okay?

Reply:
That would probably work, although at that point you've made a single use variable, so just do `ctx.send("")`

Reply:
Not sure if `ctx.send()` followed by `return` is best practice but seems like the cleanest way of sending and returning.